src/js/events/ChronosActions.js

```diff
@@ -133,6 +134,26 @@ const ChronosActions = {
     });
   },
 
+  updateJob: function (jobID, data) {
```

Comment:
The endpoint is documented as follows, but I'm fine to use a different name if you think "editJob" works better.

> Update a job. This endpoint will overwrite the complete job, it will not support PATCH-like updates.

What do you think?